bucket.go

```diff
@@ -175,15 +175,17 @@ func (b *Bucket) CreateBucket(key []byte) (*Bucket, error) {
 	var value = bucket.write()
 
 	// Insert into node.
-	key = cloneBytes(key)
-	c.node().put(key, key, value, 0, common.BucketLeafFlag)
+	// Using new variable here is important: otherwise `key` param
+	// is marked as leaking and cannot be allocated on stack.
```

Comment:
The change makes sense to me, but I wouldn't say `important`.
```suggestion
	// Tip: Use a new variable `newKey` instead of reusing the existing `key` to prevent
	// it from being marked as leaking, and accordingly cannot be allocated on stack.
```

Reply:
Fixed.